parcels/_version.py

```diff
@@ -13,9 +14,11 @@
             .decode("ascii")
             .strip()
         )
-    except subprocess.SubprocessError as e:
-        e.add_note(
+    except subprocess.SubprocessError:
+        warnings.warn(
             "Looks like you're trying to do a development install of parcels. "
             "This needs to be in a git repo so that version information is available. "
+            "Setting version to 'unknown'.",
+            stacklevel=2,
         )
-        raise e
+        version = "unknown"
```

Comment:
I thought that a warning here would be less obtrusive than erroring completely, since version info is only important for particlefile metadata.

Not that easy to test, but also not really something important to support anyway.